### PR TITLE
Update PyWeek.org.xml

### PR DESCRIPTION
**Pull request by tamoor-cliqz**

### CODE REVIEW

**Comment by J0WI** [comment]
@tamoor-cliqz This rule need to be rebased completely.
The cert is only valid for `^` and mismatches on ` media`, `pyggy` and `www`.
cc @luciancor

**Comment by luciancor** [comment]
@J0WI not sure if I understand what you mean. I removed the redirect rule which does not make sense. Can you please take a look and let me know what you mean by rebase?

**Comment by J0WI** [comment]
All the comments in this ruleset make no sense now. Can you update/remove them? I also posted some additional non-working subdomains, would be great if you can add them too.

**Comment by luciancor** [comment]
One more try :)

File: src/chrome/content/rules/PyWeek.org.xml [hunk 1]
```diff
@@ -1,21 +1,18 @@
 <!--
 	^pyweek.org: Dropped
```
Comment (J0WI) [inline src/chrome/content/rules/PyWeek.org.xml]: This comment is obsolete

File: src/chrome/content/rules/PyWeek.org.xml [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
-<ruleset name="PyWeek.org">
 
-	<!--	Direct rewrites:
-				-->
-	<target host="www.pyweek.org" />
+<ruleset name="PyWeek.org">
 
-	<!--	Complications:
-				-->
+	<!-- <target host="www.pyweek.org" /> Expired Certificate
```
Comment (J0WI) [inline src/chrome/content/rules/PyWeek.org.xml]: This comment is redundant to the one on the top

File: src/chrome/content/rules/PyWeek.org.xml [hunk 3]
```diff
@@ -1,21 +1,18 @@
 <!--
 	^pyweek.org: Dropped
 
+	non-working subdomains:
```
Comment (J0WI) [inline src/chrome/content/rules/PyWeek.org.xml]: Could you please describe what kind of error they have?

**Comment by gloomy-ghost** [comment]
superseded by #12945

**Pull request closed without merging**